bootstrap/bootstrap.sh

```diff
@@ -154,6 +154,10 @@ check_program xxd
 check_program fold
 check_program dig
 
+# Trim trailing .git from repositories
+# shellcheck disable=SC2001
+GITHUB_REPOSITORY_PARAM=$(echo "$GITHUB_REPOSITORY_PARAM" | sed 's/\.git$//')
```

Comment:
Spellcheck isn't that wrong:
```suggestion
GITHUB_REPOSITORY_PARAM=$(echo "$GITHUB_REPOSITORY_PARAM%.git")
```